### PR TITLE
Make SNB load work on jenkins

### PR DESCRIPTION
**Pull request by aelred**

### CODE REVIEW

**Comment by aelred** [comment]
We can consider this PR fit to go in once it passes the "load" step for SNB (not validate).

**Comment by aelred** [comment]
@fppt and @pluraliseseverythings we're seeing this happen:

```
2017-09-25 09:55:45,554 [redisq-task-manager-7] ERROR a.g.e.t.m.r.RedisTaskQueueConsumer - TaskId{value=3967b2f7-e633-42d9-a53e-6d4594953b68} could not be completed successfully
ai.grakn.exception.GraknTxOperationException: Relationship {E3ubrl-3tea0-f11-rvwdk} is missing the owner
	at ai.grakn.exception.GraknTxOperationException.missingOwner(GraknTxOperationException.java:302)
	at ai.grakn.kb.internal.concept.RelationshipEdge.lambda$null$4(RelationshipEdge.java:71)
	at java.util.Optional.orElseThrow(Optional.java:290)
	at ai.grakn.kb.internal.concept.RelationshipEdge.lambda$new$5(RelationshipEdge.java:71)
	at ai.grakn.kb.internal.cache.Cache.get(Cache.java:74)
	at ai.grakn.kb.internal.concept.RelationshipEdge.owner(RelationshipEdge.java:173)
	at ai.grakn.kb.internal.GraknTxAbstract.copyRelation(GraknTxAbstract.java:948)
	at ai.grakn.kb.internal.GraknTxAbstract.copyRelation(GraknTxAbstract.java:909)
	at ai.grakn.kb.internal.GraknTxAbstract.lambda$fixDuplicateResources$17(GraknTxAbstract.java:878)
	...
	at ai.grakn.kb.internal.GraknTxAbstract.fixDuplicateResources(GraknTxAbstract.java:878)
	at ai.grakn.engine.postprocessing.PostProcessingTask.runPostProcessingMethod(PostProcessingTask.java:126)
	...
	at ai.grakn.redisq.consumer.RedisqConsumer.process(RedisqConsumer.java:47)
	at ai.grakn.redisq.consumer.RedisqConsumer.process(RedisqConsumer.java:22)
	at ai.grakn.redisq.Redisq.lambda$execute$4(Redisq.java:266)
	...
```

**Comment by aelred** [comment]
@fppt this makes it to validation now, so I'd suggest a review + merge if all good.

...then we can think about getting validation working :/

File: Jenkinsfile [hunk 1]
```diff
@@ -72,7 +72,8 @@ def withPath(String path, Closure closure) {
 
 node {
     //Only run validation master/stable
-    if (env.BRANCH_NAME in ['master', 'stable']) {
+    // TODO: don't merge this change
+    if (env.BRANCH_NAME in ['master', 'stable'] || true) {
```
Comment (fppt) [inline Jenkinsfile]: I am conflicted now. I want to merge but this comment tells me not to.

Comment (aelred) [reply Jenkinsfile]: more-recent past felix has corrected less-recent past felix's mistake

**Comment by aelred** [comment]
@fppt ready now 👍

**Comment by aelred** [comment]
!rtg - something happened who knows

**Pull request merged**